Tests/test_SeqUtils.py

```diff
@@ -1,8 +1,9 @@
 # Copyright 2003 by Iddo Friedberg.  All rights reserved.
 # Copyright 2007-2009 by Peter Cock.  All rights reserved.
-# This code is part of the Biopython distribution and governed by its
-# license.  Please see the LICENSE file that should have been included
-# as part of this package.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
```

Comment:
Have you reviewed the contributors to this file? See #898

Reply:
Didn't realize the legal complications, will revert to the original license for the file

Reply:
Great - I though maybe you'd edited the wrong file (``Tests/test_ctd.py`` needed the dual license block).